### ISSUE TITLE
Automated TC message  when a job offer is accepted  - candidate becomes member of Pathway Club

### ISSUE THREAD

**Issue by camerojo** [talentbeyondboundaries/tbbtalentv2#931]

**Comment by cazcam34** [comment]
Do we want to notify chats are are older (e.g Candidate Prospect/JobCreatorSourcePartner) as there are now newer chats created where this information can be shared with the same members (e.g. Candidate Recruiting which contains the same members as older chats - destination, source and candidate). Basically do we notify in one place where everyone can see it, or do we want to notify on all chats that were maybe relevant to this recruitment process?

I beleive this should notify the following chats...
- Candidate Recruiting

**Comment by cazcam34** [comment]
Wait on PR #1014 to be merged into staging as this is a stage change post, so will tie in. If stage change is to stage accepted then send posts.

**Comment by cazcam34** [comment]
Waiting PR #1048 to be merged as it has code that I will extending on for this PR.

**Comment by cazcam34** [comment]
Link pathway club in post content
